common/djangoapps/edeos/models.py

```diff
@@ -10,19 +10,28 @@ class UserSocialLink(models.Model):
     """
     # LINKEDIN = 'linkedin'
     FACEBOOK = 'facebook'
+    SKYPE = 'skype'
+    VK = 'vk'
+    TELEGRAM = 'telegram'
 
     PLATFORM_CHOICES = (
         # (LINKEDIN, ugettext_noop('LinkedIn')),
         (FACEBOOK, ugettext_noop('Facebook')),
+        (SKYPE, ugettext_noop('Skype')),
```

Comment:
Because on Skillonomy even names are translated to russian quite often.
> In this case they were not translated before adding to database. But at the same time they were added to *.po files.
> In this example ugettext or ugettext_lazy functions should be used only after extracting messages from database.
https://stackoverflow.com/questions/43084652/whats-the-purpose-of-ugettext-noop-in-django